### ISSUE TITLE
Neovim: allow users to turn off "--color=never"

### ISSUE THREAD

**Issue by JakobBruenker** [ndmitchell/ghcid#215]
Right now, in the function s:ghcid, "--color=never" is hardcoded as a command, and it cannot be overwritten by typing "--color=always" in the `g:ghcid_command`, as far as I can tell.It would be nice if

**Comment by ndmitchell** [comment]
Since `ghcid-0.7` color codes were supported by `ghcid` itself. Are you referring to extra parsing that Neovim or the Neovim Ghcid plugin are doing?

**Comment by cloudhead** [comment]
Yeah - I'm talking about extracting error messages out of the ghcid output.

**Comment by drewboardman** [comment]
Any update on this? Would be a nice feature.

**Comment by cloudhead** [comment]
The fix would be to allow you to overwrite the `--color` argument, but that would break parsing, as I said - so it's not really an acceptable fix in my opinion. I don't have the time to build a system that would allow color output *and* adding the errors into quickfix, though it should be possible using something like `tee`.
